src/ipc/ipc4/handler.c

```diff
@@ -843,11 +843,13 @@ static int ipc4_module_process_d0ix(struct ipc4_message_request *ipc4)
 		return IPC4_INVALID_RESOURCE_ID;
 	}
 
-	/* only module 0 can be used to set d0ix state */
+	/* only module 0 can be used to set d0ix state.
```

Comment:
this comment seems to refer to the above code - it should be moved above line 841

Reply:
thanks, done